cirq-core/cirq/protocols/decompose_protocol.py

```diff
@@ -263,6 +238,9 @@ def try_op_decomposer(val: Any, decomposer: Optional[OpDecomposer]) -> Decompose
     return output
 
 
+# pylint: disable=function-redefined
+
+
```

Comment:
Nit: this looks like it's no longer needed.

Reply:
This is still needed for the other overloaded methods in this file.